frontend/src/routes/Home/Search/components/Modals/DeleteResourceModal.test.tsx

```diff
@@ -217,17 +217,22 @@ describe('DeleteResourceModal', () => {
             expect(submitButton).toBeTruthy()
             userEvent.click(submitButton)
 
-            // Wait for delete resource requesets to finish
-            await waitForNocks([deleteResourceNock])
+            // // Wait for delete resource requesets to finish
```

Comment:
@jeswanke Do you plan to remove it?

Reply:
@chenz4027 hey Maggie--good question--I just moved it to the bottom of the test---this test has been randomly ending before this nock has been finished--not sure what funkiness is going on with where that nock wait was located

Reply:
Ah I see, thanks John for the explanation :)